lib/utils/s3-utils.ts

```diff
@@ -122,3 +122,12 @@ export const getSignedUrlForS3 = (
   const url = s3.getSignedUrl('getObject', params);
   return url;
 };
+
+/**
+ * Get the file name of a file stored in S3 from its object key
+ * @param objectKey S3 object key
+ * @returns name of file stored in S3
+ */
+export const getFileName = (objectKey: string): string | undefined => {
```

Comment:
thanks for adding this! If you didn't already, can you do a quick find for where we use split('/').at(-1) for file names and replace it with this function

Reply:
I left it out originally because it was irrelevant to this ticket (didn't want to further clutter the file diffs) but I can add it in now that you've reviewed it

Reply:
Fixed